allTests.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name = "All Tests">
```

Comment:
so you are able for all tests to use the previous approach, use this approach for such suite as "Smoke suite" and choose few test with this approach